test/api/cbg/input.json

```diff
@@ -32,5 +32,13 @@
         "units": "mg/dl",
         "deviceId": "tools",
         "uploadId": "tools"
+    },
+    {
+        "time": "2015-12-21T11:23:08Z",
+        "type": "cbg",
+        "value": 69,
+        "units": "mg\/dL",
+        "deviceId": "DexHealthKit_Dexcom:com.dexcom.Share2:3.0.4.17",
```

Comment:
I'm not aware of them sending an empty string for `deviceId`. They sent an empty POST body and had problems, but that's a larger issue...